synfig-studio/src/player/main.cpp

```diff
@@ -83,6 +84,8 @@ int main(int argc, char **argv)
 		std::cout << std::endl;
 		std::cout << "usage: " << std::endl;
 		std::cout << "  " << commandname << " <file.sif|file.sifz> <renderer>" << std::endl;
```

Comment:
Please replace " <file.sif|file.sifz> <renderer>" to " <file.sif|file.sifz> <renderer> [--benchmark]"